CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ### 1.2.1 (Next)
 
 * [#82](https://github.com/dblock/iex-ruby-client/pull/82): Added `config.referer` to set HTTP `Referer` header. This enables IEX's "Manage domains" domain locking for tokens - [@agrberg](https://github.com/agrberg).
+* [#70](https://github.com/dblock/iex-ruby-client/pull/70): Add type safe cast when ref_data_isin parameter is string - [@rodolfobandeira](https://github.com/rodolfobandeira).
```

Comment:
maybe 'Added support for `ref_data_isin` taking a single `String` parameter", it's unclear what a type safe cast means here

Reply:
Totally agree.
Done